Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/duckduckgo/content-scope-scripts",
         "state": {
           "branch": null,
-          "revision": "ed10de3485f454ca59259266a0b3942bd21b43db",
-          "version": "3.4.1"
+          "revision": "801b7f23476f797c6eaa72b070e6c80abb82801a",
```

Comment:
This change seems unwanted, but it likely isn’t :) Since we started including Package.resolved in git, some automations still need to catch up and resolve package dependencies after updating Package.swift. Here a content scope scripts automation bumped the version but didn’t resolve dependencies, causing a mismatch. I think this should be checked in 👍